tests/test_measurements.py

```diff
@@ -1 +1,36 @@
 """Tests related to petab.measurements"""
+import petab
+import pytest
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'pytest'](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4927042)